web-client/public/js/update-app.js

```diff
@@ -345,5 +348,22 @@ export const updateApp = grnState => {
         updatetoGridLayout();
     }
 
+/* Node Coloring
+
+    if (grnState.nodeColoring.avgTopDataset) {
+        $(AVG_REPLICATE_VALS_TOP_MENU + " span").addClass("glyphicon-ok");
+        $(AVG_REPLICATE_VALS_TOP_MENU).prop("checked", "checked");
+        $(AVG_REPLICATE_VALS_TOP_SIDEBAR).prop("checked", "checked");
+        updaters.renderNodeColoring();
+    } else {
+        $(AVG_REPLICATE_VALS_TOP_MENU + " span").removeClass("glyphicon-ok");
+        $(AVG_REPLICATE_VALS_TOP_MENU).removeProp("checked");
+        $(AVG_REPLICATE_VALS_TOP_SIDEBAR).removeProp("checked");
+        updaters.renderNodeColoring();
+    }
+
     refreshApp();
+
+*/
+
```

Comment:
Maybe plan to eliminate comments soon after merging? I know they have some use while in the middle of things, but beyond that, let GitHub do the change-tracking instead.